doc/tutorials/doc_examples/experiment_doc_examples/exp.py

```diff
@@ -17,10 +17,13 @@
 # Generate the output directory
 exp.generate(standalone_database, model, overwrite=True)
 
+# Preview the experiment
+exp.preview(standalone_database, model, verbosity_level="debug")
```

Comment:
Do you want your example to send strings or would you prefer to have them see `verbosity_level=Verbosity.DEBUG`?

Reply:
resolved.